sklearn/linear_model/_perceptron.py

```diff
@@ -97,20 +97,24 @@ class Perceptron(BaseSGDClassifier):
 
     Attributes
     ----------
+    classes_ : array of shape (n_classes,)
+        The unique classes labels.
+
     coef_ : array, shape = [1, n_features] if n_classes == 2 else [n_classes,\
             n_features]
         Weights assigned to the features.
 
     intercept_ : array, shape = [1] if n_classes == 2 else [n_classes]
         Constants in decision function.
 
+    loss_function_ : concrete LossFunction
```

Comment:
I think I have reviewed this before.

Was it you by any chance who made this change in an earlier PR?
Is this the same PR or a new one?

If it is a new one, did you restart from the freshest version of scikit-learn or from the changes you had already done?